src/config/songConfig.ts

```diff
@@ -0,0 +1,57 @@
+import z from "zod";
+
+// Define the number of songs based on the environment variables present
+const getSongConfigCount = (): number => {
+  let count = 1;
+  while (process.env[`SONG_${count}_URL`]) {
+    count++;
+  }
+  return count - 1;
```

Comment:
lol why start with 1 only to remove it later?
```suggestion
  let count = 0;
  while (process.env[`SONG_${count}_URL`]) {
    count++;
  }
  return count;
```

Reply:
changed this counter to start with `0` and updated the doc accordingly.

Reply:
🤔 I can't see the change here... am I missing something?

Reply:
just pushed the commit with all the changes 😅